src/solvers/solvers_parabolic.jl

```diff
@@ -28,4 +33,34 @@ struct ViscousFormulationLocalDG{P}
     penalty_parameter::P
 end
 
+"""
+    ViscousFormulationLocalDG()
+
+The minimum dissipation local DG (LDG) flux from "An Analysis of the Minimal Dissipation Local 
+Discontinuous Galerkin Method for Convection–Diffusion Problems" by Cockburn and Dong (2007). 
+This scheme corresponds to an LDG parabolic "upwinding/downwinding" but no LDG penalty parameter. 
+Cockburn and Dong proved that this scheme is still stable despite the zero penalty parameter. 
+
+- Cockburn and Dong (2007)  
+  An Analysis of the Minimal Dissipation Local Discontinuous 
+  Galerkin Method for Convection–Diffusion Problems.
+  [DOI: 10.1007/s10915-007-9130-3](https://doi.org/10.1007/s10915-007-9130-3)
+"""
+ViscousFormulationLocalDG() = ViscousFormulationLocalDG(nothing)
+
+# Here, the flux is {{f}} + beta * [[f]], where beta is the LDG "switch", 
+# which we set to -1 on the left and +1 on the right in 1D. The sign of the 
+# jump term should be opposite that of the sign used in the divergence flux. 
+# This is equivalent to setting the flux equal to `u_ll` for the gradient,
+# and `u_rr` for the divergence. 
+function flux_parabolic(u_ll, u_rr, ::Gradient, mesh::TreeMesh, equations,
+                        parabolic_scheme::ViscousFormulationLocalDG)
+    return u_ll # Use the upwind value for the gradient interface flux
+end
+
+function flux_parabolic(u_ll, u_rr, ::Divergence, mesh::TreeMesh, equations,
+                        parabolic_scheme::ViscousFormulationLocalDG)
+    return u_rr # Use the downwind value for the divergence interface flux
+end
```

Comment:
As far as I understand, this means that the `penalty_parameter` is never used, is it?

Reply:
Ah - yes, it does not appear to be implemented for `TreeMesh` (it was for `DGMulti` solvers). 

The penalty parameter is imposed in the divergence interface flux; I can add it in this PR as well or leave it until the next one. Any preference?

Reply:
I'm fine having it in another PR since this one is already big enough. Could you please just update the LDG docstring accordingly?